# target frame id
# string
target_frame_id: /map
table_frame_id: /bookcase_bottom_link

height_min: 0.01
height_max: 0.5

table_dimensions: [0.45, 1.55, 0.8]

